packages/remark-parse/lib/util/get-indentation.js

```diff
@@ -25,7 +26,10 @@ function indentation(value) {
       indent = Math.floor(indent / size) * size
     }
 
-    stops[indent] = index
+    while (lastIndent < indent) {
```

Comment:
For spaces (i.e. with `indent - lastIndent == 1`), this is equivalent to just `stops[indent]=index; lastIndent=indent`

For tabs, this will create entries in `stops` for all intermediate indents since the last one. For example, if the last indentation point is `1` (a space), and the next indentation point is `5` (a tab follows the initial space), this will add stops 2, 3, 4 and 5 pointing to character 1 (the tab).